stetho/src/main/java/com/facebook/stetho/common/android/FragmentActivityAccessor.java

```diff
@@ -6,10 +6,9 @@
 
 import javax.annotation.Nullable;
 
-public interface FragmentActivityAccessor {
+public interface FragmentActivityAccessor<
+    FRAGMENT_ACTIVITY extends Activity,
+    FRAGMENT_MANAGER> {
   @Nullable
-  public Object getFragmentManager(Activity fragmentActivity);
-
-  @Nullable
-  public Object getSupportFragmentManager(Activity fragmentActivity);
+  public FRAGMENT_MANAGER getFragmentManager(FRAGMENT_ACTIVITY activity);
```

Comment:
I believe I preserved the original behaviour by having two accessors,
rather than one with two methods.  See the caller code which previously
invoked getSupportFragmentManager.